tests/js/spec/components/organizations/timeRangeSelector/dateSummary.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import {cleanup, mountWithTheme, screen} from 'sentry-test/reactTestingLibrary';
+
+import DateSummary from 'app/components/organizations/timeRangeSelector/dateSummary';
+
+const start = new Date('2017-10-14T02:38:00.000Z');
+const end = new Date('2017-10-17T02:38:00.000Z'); // National Pasta Day
+
+describe('DateSummary', () => {
+  it('renders', async () => {
+    const {container} = mountWithTheme(<DateSummary start={start} end={end} />);
+    expect(container).toSnapshot();
+  });
+
+  it('does not show times when it is midnight for start date and 23:59:59 for end date', () => {
+    mountWithTheme(<DateSummary start={start} end={end} />);
+    // Search by year because day may change depending on timezone
+    expect(screen.getAllByText(/Oct.+2017/)[0].childElementCount).toBe(1);
+    cleanup();
```

Comment:
should we use [rerender](https://testing-library.com/docs/react-testing-library/api/#rerender) instead?